pym/bob/cmds/build.py

```diff
@@ -1068,3 +1069,107 @@ def doClean(argv, bobRoot):
         if not args.dry_run:
             removePath(d)
 
+def doQueryPath(argv, bobRoot):
+    # Local imports
+    from string import Formatter
+
+    # Configure the parser
+    parser = argparse.ArgumentParser(prog="bob query-path",
+                                     description="Lists all packages and their last build directory. If a package has never been built, no output is shown for that package.")
```

Comment:
It would really be nice to have some short paragraph in the help text explaining the available output formats. Otherwise nobody knows the options without reading the source code. There are some expamples in `doClean()` and `doQuerySCM()` how to that.